baselines/pFedHN/pFedHN/models.py

```diff
@@ -0,0 +1,222 @@
+"""Implement the HyperNetwork and TargetNetwork for pFedHN."""
+import ssl
+from collections import OrderedDict
+
+import torch.nn.functional as F
+from torch import nn
+from torch.nn.utils import spectral_norm
+
+# Disable protected member warnings for SSL
+# pylint: disable=protected-access
+
+ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Why is this needed?

Reply:
Here this is a ssl command which will  represents an "unverified" SSL context, which is created with certificate verification turned off. In other words, it allows HTTPS connections to be made without verifying the SSL certificates of the remote server.
We tried running the code by removing the above line. But initially we faced an error. 
We will verify if its needed or not by updating all the packages and decide upon keeping it or removing it.